app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with(model: comment, url: [project, task, comment]) do |form| %>
+  <% if comment.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(comment.errors.count, "error") %> prohibited this project from being saved:</h2>
+
+      <ul>
+        <% comment.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.label :content %>
+  <%= form.text_field :content %>
+  <%= form.hidden_field :user_id, value: current_user.id %>
```

Comment:
Remove it. We don't need this information from form, we have current user in controller. It is unsafe